server/server.js

```diff
@@ -4,13 +4,20 @@
 
 const express = require('express');
 const path = require('path');
+const bodyParser = require('body-parser');
 
 const config = require('./config');
 const httpsServer = require('./https_server/https_server');
 const redirectServer = require('./https_server/redirect_server');
 
 const app = express();
+
 app.use(express.static(path.join(__dirname, '../client')));
+app.use(bodyParser.json({ limit: '10mb' }));
```

Comment:
I'm not sure if this limit is really needed in this branch.
IMHO, it's better to add this option if we really need this.
And if we need this right now, please leave a comment for this.